### ISSUE TITLE
CustomPaint control

### ISSUE THREAD

**Issue by FeodorFitsner** [flet-dev/flet#205]
https://www.raywenderlich.com/25237210-building-a-drawing-app-in-flutter

**Comment by mimavox** [comment]
Yes please! Considering using Fleet as the go-to GUI tool for my introductory Python course, and a painter would be an invaluable tool for my students to play around with.

**Comment by FeodorFitsner** [comment]
Sure, my only concern is to how to deal with a canvas's "size" property which is available on Flutter side in `paint()` method and, presumably, be useful on a Python (server) side. 😕

**Comment by FeodorFitsner** [comment]
Maybe we could offer two options:
1) Flet developer knows the size of the canvas (say, it's a widget with page's width/height or a widget with a fixed size) and can provide drawing instructions right away.
2) Flet developer doesn't know canvas size, so their provide drawing instructions in `custom_paint.on_paint` event handler which brings canvas size in its event object.

**Comment by FeodorFitsner** [comment]
https://flet.dev/blog/canvas
